cypress/fixtures/single-evidence-request-response.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "createdAt": "2020-11-30T15:34:12.299Z",
+  "resident": {
+    "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+    "name": "Namey McName",
+    "email": "frodo@bagend.com",
+    "phoneNumber": "+447123456780"
```

Comment:
should we have a real phone number?

Reply:
AFAIK it's a fake phone number (+447**12345678**)